src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
```

Comment:
Remove unnecessary empty line

Reply:
wow, this reviewed piece of code is something different that I have in my final version.  I don't have any h1 in the body.

Reply:
red highlight means that this line was deleted